src/Interfaces/PaymentServiceInterface.php

```diff
@@ -60,8 +63,8 @@ public function performAuthorization(
         Authorization $authorization,
         $paymentType,
         $customer = null,
-        Metadata $metadata = null,
-        Basket $basket = null
+        Metadata      $metadata = null,
+        Basket        $basket = null
```

Comment:
I don´t understand this formatting :)

Reply:
This is automatic formatting from cs-fixer to align parameter.